twilight-gateway/README.md

```diff
@@ -114,15 +115,13 @@ There are a few additional examples located in the
 
 [`CryptoProvider::install_default`]: https://docs.rs/rustls/latest/rustls/crypto/struct.CryptoProvider.html#method.install_default
 [`aws-lc-rs`]: https://crates.io/crates/aws-lc-rs
-[`flate2`]: https://crates.io/crates/flate2
```

Comment:
Why is this removed? (same for the other link, the references still exist, so I don't see why?)